templates/docker-compose.template

```diff
@@ -95,8 +93,6 @@ services:
       - miner-storage:/var/data
       - watchdog-log:/var/watchdog
       - nebra:/var/nebra
-    ports:
```

Comment:
why move to end?

Reply:
No specific reason :smile: Just appended. Now noved up.